apps/ledger-live-desktop/tests/specs/speculos/delegate.spec.ts

```diff
@@ -47,17 +48,120 @@ const validators = [
   },
 ];
 
-for (const account of e2eDelegationAccounts) {
-  test.describe("Delegate", () => {
+test.describe("Delegate flows", () => {
+  test.beforeAll(async () => {
+    process.env.ENABLE_TRANSACTION_BROADCAST =
+      new Date().getDay() === 1 && isRunningInScheduledWorkflow() ? "true" : "false";
```

Comment:
Should be `new Date().getDay() === 1 && isRunningInScheduledWorkflow() ? "1" : "0";` no ?

Reply:
I think both work tbh, will update still

Reply:
https://github.com/LedgerHQ/ledger-live/blob/3eee97630e774b2ad38b8427f0328d0c77226fcb/apps/ledger-live-desktop/tests/fixtures/common.ts#L156

We're checking if  equals "1"